docs/proposals/form-metadata.md

```diff
@@ -0,0 +1,128 @@
+# Add UI metadata to catalog requests
+
+## Introduction
+Support better provision user experience for APBs by giving additional metadata about APB parameters.
+
+## Problem Description
+Currently, APB parameters are shown in random order and without any kind of grouping.  the OpenServiceBroker API utilizes json-schema data format and does not include any UI metadata.  As such, the UI cannot customize the experience and users are confronted with a bad experience which gets worse as the number of parameters increases. 
+
+## Implementation Details
```

Comment:
Is this WIP? 

I don't think this change will impact a lot of the broker, but a quick rundown of where changes may need to be made might be nice.